src/main/java/com/conveyal/r5/transit/TransitLayer.java

```diff
@@ -184,6 +192,9 @@ public void loadFromGtfs (GTFSFeed gtfs, LoadLevel level) throws DuplicateFeedEx
             // This is only used while building the TransitNetwork to look up StopTimes from the same feed.
             indexForUnscopedStopId.put(stop.stop_id, stopIndex);
             stopIdForIndex.add(scopedStopId);
+            // intern zone IDs to save memory
```

Comment:
gtfs-lib might already intern every string field it reads - I guess this doesn't hurt but might not be necessary.

Reply:
When reading zone_id and parent_station, loadOneRow of stops.txt, gtfs-lib indeed uses a Deduplicator which "does the same thing as String.intern"